index.js

```diff
@@ -619,7 +719,22 @@ Read more on https://git.io/JJc0W`);
    */
 
   option(flags, description, fn, defaultValue) {
```

Comment:
It's pretty much the same as `requiredOption`.
I thought it would be better to make it a function like the original code, but what do you think?

Reply:
It was much smaller when I first did it so seemed reasonable not sharing code, but I had left out the old regexp code! I'll take another look.

Reply:
I changed `.requiredOption()` to call `.option()` to do most of the work. Means using knowledge that added to end of `options` array, but nicely consistent with `requiredOption` just being `option` plus mandatory.